h5p-bildetema/src/components/PrintButton/PrintButton.module.scss

```
@@ -19,3 +19,49 @@
     text-decoration: underline;
   }
 }
+
+.printWordsWrapper {
+  display: none;
+}
+
+.printDropdownContent {
+  display: none;
+  position: absolute;
+  flex-direction: column;
+  margin-top: 1rem;
+  background-color: white;
+  min-width: 3rem;
+  box-shadow: hsl(206 22% 7% / 35%) 0 10px 38px -10px,
+    hsl(206 22% 7% / 20%) 0 10px 20px -15px;
+  z-index: $fav-language-selector-z-index;
+  padding: 25px;
+  color: black;
+}
+
+.show {
+  display: flex;
+}
+
+.printDropdownElementWrapper {
+  cursor: pointer;
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  border-bottom: 1px solid lightgrey;
```

Maybe not show border-bottom on the last element (place underneath transition-duration perhaps)

```suggestion
  &:not(:last-child) {
    border-bottom: 1px solid lightgrey;
  }
```